src/source/PeerConnection/SessionDescription.c

```diff
@@ -582,6 +610,42 @@ STATUS populateSessionDescriptionDataChannel(PKvsPeerConnection pKvsPeerConnecti
     return retStatus;
 }
 
+BOOL isPresentInRemote(PKvsRtpTransceiver pKvsRtpTransceiver, PSessionDescription pRemoteSessionDescription) {
+
+    PCHAR remoteAttributeValue, end;
+    UINT32 remoteTokenLen;
+
+    MEDIA_STREAM_TRACK_KIND localTrackKind = pKvsRtpTransceiver->sender.track.kind;
+
+    for (int i = 0; i < pRemoteSessionDescription->mediaCount; i++) {
+        PSdpMediaDescription pRemoteMediaDescription = &pRemoteSessionDescription->mediaDescriptions[i];
+        remoteAttributeValue = pRemoteMediaDescription->mediaName;
+        
+        if ((end = STRCHR(remoteAttributeValue, ' ')) != NULL) {
+            remoteTokenLen = (end - remoteAttributeValue);
+        } else {
+            remoteTokenLen = STRLEN(remoteAttributeValue);
+        }
+
+        switch (localTrackKind) {
+            case MEDIA_STREAM_TRACK_KIND_AUDIO:
+                if ( remoteTokenLen == STRLEN(MEDIA_SECTION_AUDIO_VALUE) && STRNCMP(MEDIA_SECTION_AUDIO_VALUE, remoteAttributeValue, remoteTokenLen) == 0 ) {
```

Comment:
Minor: STRLEN will be a run-time evaluated. This is OK here due to no perf issues. However, I would instead us ARRAY_SIZE(MEDIA_SECTION_AUDIO_VALUE) - 1 which would then be a compile-time evaluated